### DISCUSSION TITLE
关于ura-core兼容性的问题

### DISCUSSION THREAD

**Discussion by DizzyDwarf3333** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#62] [Q&A]
请问作者大大能为ura-core出一个像umamusume-localify修改版那样的兼容模式吗？
umamusume-localify的配置文件还是会导致在根目录无法安装其他插件
而ura-core没有这个问题
麻烦大大了

**Comment by EtherealAO** [comment]
为ura-core出一个兼容模式？不是已经有了吗

**Reply by DizzyDwarf3333** [reply]
搞明白如何用了，谢谢
